.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "sourceMaps": "true",
-  "stage": 2
-}
-
+  "sourceMaps": true,
+  "stage": 2,
+  "optional": ["runtime"]
```

Comment:
Added `runtime` and `babel-runtime` in dependencies...

I really wish we where moved to node 4.0 so we could use generators with bluebird :)
But we'll get there after this round which will do major clean up !! I made so much bad copy/paste code in bin/ folders..